samples/bluetooth/hci_vs_scan_req/overlay-bt_ll_sw_split.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_BT_CTLR_VS_SCAN_REQ_RX=y
+CONFIG_BT_CTLR_ADVANCED_FEATURES=y
+CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL=y
```

Comment:
Why do we need dynamic power control for this sample?

Reply:
It's required in our app module control of Tx power per role, not in this sample: removed.